### PR TITLE
Load the desktop app with code splitting turned on

### PR DESCRIPTION
**Pull request by blowery**
Try building and loading the desktop app with code splitting turned on, better mirroring how Calypso typically builds and loads. 

Build times should be much better.

Pairs with https://github.com/Automattic/wp-calypso/pull/33585

### CODE REVIEW

**Comment by codebykat** [comment]
I dunno, I still can't `make build`. On this branch I get:

`ERROR in ./desktop/index.js
Module build failed: Error: Cannot find module '@babel/preset-typescript' from '/Users/kat/code/wp-desktop'`

**Review by loremattei — APPROVED** [review]
This looks good to me. 
I tested `make build` and `make dev-server`/`make dev` and they completed without errors on my machine. 

Also, the build time and the app startup time have improved noticeably on my system! Thanks @blowery  :-) :-)

**Comment by blowery** [comment]
@codebykat sounds like you might need to run `npm i` to pick up the new babel bits, or the calypso submodule is still on the old version and not using typescript yet. `git submodule update` might help there.

@loremattei woo, thanks!

**Comment by codebykat** [comment]
TY @blowery, one of those two things was it. `make build` finishes for me in 16 min now and I'm also able to `make dev-server / make dev`.

File: Makefile [hunk 1]
```diff
@@ -59,7 +59,7 @@ dev-server: checks
 
 	$(MAKE) desktop$/config.json CONFIG_ENV=$(CONFIG_ENV)
 
-	@npx concurrently -k \
+	npx concurrently -k \
```
Comment (blowery) [inline Makefile]: should revert this stuff

**Pull request merged**